### PR TITLE
первая схема

### PR DESCRIPTION
**Pull request by Alex8185**
Учусь, все пригодится, повторение - мать учения.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Alexis220382) [inline .idea/vcs.xml:1]: seems this file should be under the exclusions as a whole folder `.idea`

File: src/main/java/by/shop/App.java [hunk 2]
```diff
@@ -8,6 +10,11 @@
 {
     public static void main( String[] args )
     {
-        System.out.println( "Hello World!" );
+        Client klient1 = new Client();
+        Celler celler1 = new Celler();
```
Comment (Alexis220382) [inline src/main/java/by/shop/App.java:14]: Lesh, correct the name, please to Seller. The same with `klient` - should be `client`
And why do you use 1 in variables. Do you have second seller and client?

File: src/main/java/by/shop/Celler.java [hunk 3]
```diff
@@ -1,4 +1,7 @@
 package by.shop;
 
-public class Celler {
+public class Celler extends Speak{
+    public void heSpeaks() {
```
Comment (Alexis220382) [inline src/main/java/by/shop/Celler.java:4]: Excellent approach!!!
But do some corrections please: 
- rename method to `speak()`;
- give opportunity to speak not only "Добрый день!";

File: src/main/java/by/shop/Client.java [hunk 4]
```diff
@@ -1,4 +1,8 @@
 package by.shop;
 
-public class Client {
+public class Client extends Speak{
+    public void heSpeaks() {
+        System.out.println("Добрый день!");
```
Comment (Alexis220382) [inline src/main/java/by/shop/Client.java:5]: The same as above👆

File: src/main/java/by/shop/Speak.java [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package by.shop;
+
+public abstract class Speak {
```
Comment (Alexis220382) [inline src/main/java/by/shop/Speak.java:3]: ?????????
Again you are using a verb as a class.
Remove it.
Can you explain why did you use that?

**Review by Alexis220382 — CHANGES_REQUESTED** [review]
Not so bad as a first schema, but need to improve some things

File: src/main/java/by/shop/Celler.java (hunk 3 above)
Comment (Alex8185) [reply src/main/java/by/shop/Celler.java:4]: 1. Renamed speak( );
2. With this problem. The client speaks out of sequence.

File: src/main/java/by/shop/Client.java (hunk 4 above)
Comment (Alex8185) [reply src/main/java/by/shop/Client.java:5]: ок.
How do you make them speak in turn?

File: src/main/java/by/shop/Speak.java (hunk 5 above)
Comment (Alex8185) [reply src/main/java/by/shop/Speak.java:3]: Rename with Dialogue.

File: src/main/java/by/shop/App.java (hunk 2 above)
Comment (Alex8185) [reply src/main/java/by/shop/App.java:14]: Eliminated

File: src/main/java/by/shop/Client.java (hunk 4 above)
Comment (Alexis220382) [reply src/main/java/by/shop/Client.java:5]: I think it should be method with parameter
`public void speak(String phrase) {
         System.out.println(phrase);
}`

File: src/main/java/by/shop/Speak.java (hunk 5 above)
Comment (Alexis220382) [reply src/main/java/by/shop/Speak.java:3]: No. Eliminate it.
Dialogue is unneeded.
...
You may provide this behaviour via interface `Speakable` and define abstract method `speak(String phrase);` but in that case you need to define abstract class Human and extend in with your classes Client and Seller. 
So, then you need implement this method in you abstract class Human only but not in classes Client and Seller.
After these manipulations))) Client and Seller can start speak)))
...

**Pull request merged**